src/server/test/db/meterTests.js

```diff
@@ -15,7 +15,7 @@ function expectMetersToBeEquivalent(expected, actual) {
 mocha.describe('Meters', () => {
```

Comment:
I think for this test and the next one it would be good to use some value for timezone to check the code more.